src/components/header.tsx

```diff
@@ -82,112 +120,131 @@ const Header: React.FC<{
   isLandingPage?: boolean;
 }> = ({ isLandingPage }) => {
   const [burgerMenuIsOpen, setBurgerMenuStatus] = useState(false);
+  const isScrollingUp = useScrollDirection() === "up";
 
   return (
-    <div className="header">
-      {isLandingPage && <MoratoriumBanner />}
-      <FocusTrap
-        active={burgerMenuIsOpen}
-        focusTrapOptions={{
-          onDeactivate: () => setBurgerMenuStatus(false),
-          clickOutsideDeactivates: true,
-        }}
-      >
-        <nav className="navbar" role="navigation" aria-label="main navigation">
-          <div className="navbar-brand">
-            <div
-              className="navbar-item is-flex-direction-column is-justify-content-center no-underline"
-              onClick={() => setBurgerMenuStatus(false)}
-            >
-              <Link to="/">
-                <img
-                  className="is-hidden-touch"
-                  src={require("../img/brand/logo.svg")}
-                  width="164"
-                  height="38"
-                  alt="JustFix"
-                />
-                <img
-                  className="is-hidden-desktop"
-                  src={require("../img/brand/logo.svg")}
-                  width="120"
-                  height="27"
-                  alt="JustFix"
-                />
-              </Link>
-              {isDemoSite && (
-                <span className="tag is-yellow">
-                  <Trans>DEMO SITE</Trans>
-                </span>
-              )}
-            </div>
-            <div
-              className="navbar-item is-size-3 has-text-black is-hidden-touch"
-              onClick={() => setBurgerMenuStatus(false)}
-            >
-              <Trans>Technology for Housing Justice</Trans>
-            </div>
-            <div
-              className="navbar-item is-paddingless is-flex-grow-1"
-              onClick={() => setBurgerMenuStatus(false)}
-            />
-            <div
-              className="navbar-item is-hidden-touch"
-              onClick={() => setBurgerMenuStatus(false)}
-            >
-              <Link to="/tools" className="button is-primary">
-                <Trans>See our tools</Trans>
-              </Link>
-            </div>
-            <div className={"navbar-item " + (burgerMenuIsOpen && "is-active")}>
-              <button
-                role="button"
-                className={
-                  "navbar-burger burger " + (burgerMenuIsOpen && "is-active")
-                }
-                aria-expanded="false"
-                onClick={() => setBurgerMenuStatus(!burgerMenuIsOpen)}
-                data-target="navbar"
+    <>
+      <div
+        className={classnames(
+          "jf-sticky-phantom is-hidden-desktop",
+          isScrollingUp && "jf-sticky"
+        )}
+      />
+      <div className={classnames("header", isScrollingUp && "jf-sticky")}>
```

Comment:
Adding the "phantom" sibling div (needed to prevent a jump when the header becomes fixed) meant the whole thing needed to go in the fragment, so the indent made this diff huge, but these are the only lines actually changed.

Reply:
oh nice, thanks for pointing that out!!